plugins/deepgram/src/stt.ts

```diff
@@ -261,12 +261,14 @@ export class SpeechStream extends stt.SpeechStream {
                   if (isFinal) {
                     this.queue.put({
                       type: stt.SpeechEventType.FINAL_TRANSCRIPT,
-                      alternatives: [alternatives[0], ...alternatives.splice(0)],
+                      // Ensures that TS knows that the array is not empty
```

Comment:
can you delete these comments? these are helpful at review time but not so much as code

Reply:
I don't agree (would have been useful to me when reading this code), but sure, removed